### PR TITLE
use RH based images

### PR DESCRIPTION
**Pull request by cooktheryan**
To support Konflux in the future the images must use RHEL based images

### CODE REVIEW

File: Containerfile.ps
[the remark this answers is not in the document]
Comment (cooktheryan) [reply Containerfile.ps:1]: @vishnoianil most likely has this answer

[the remark this answers is not in the document]
Comment (vishnoianil) [reply Containerfile.ps:1]: interesting, why does it shows as a binary, it's another container file for building image for the path service. here -> https://github.com/instructlab/ui/blob/main/Containerfile.ps

[the remark this answers is not in the document]
Comment (Gregory-Pereira) [reply Containerfile.ps:1]: Yeah makes sense. Just re-implement it as a containerfile @cooktheryan. Or is it that Github is recognizing it as a binary when its not?

[the remark this answers is not in the document]
Comment (cooktheryan) [reply Containerfile.ps:1]: @Gregory-Pereira @vishnoianil i'm wondering it is the extension. Looking at the initial PR it appears this way as well

https://github.com/instructlab/ui/commit/b2bec650b317a985c3a1ecab08ff9e5802eee479#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52L42

[the remark this answers is not in the document]
Comment (cooktheryan) [reply Containerfile.ps:1]: @vishnoianil ill try to keep them as part of this PR just to be less impactful with needing to land the changes immediately together. Will have something after a bit

**Comment by cooktheryan** [comment]
Hold until #334

File: Containerfile.ps
[the remark this answers is not in the document]
Comment (Gregory-Pereira) [reply Containerfile.ps:1]: or we could rename it ps.Containerfile

[the remark this answers is not in the document]
Comment (cooktheryan) [reply Containerfile.ps:1]: didn't even think about that 🤦

**Comment by cooktheryan** [comment]
@brents-pet-robot @nerdalert ready for review

**Review by vishnoianil — APPROVED** [review]
LGTM

**Comment by cooktheryan** [comment]
@vishnoianil this can merge. Tested on a clean AWS instance with no login

```
[root@ip-172-31-37-56 ~]# podman pull registry.access.redhat.com/ubi9/nodejs-22:9.5-1730543890
Trying to pull registry.access.redhat.com/ubi9/nodejs-22:9.5-1730543890...
Getting image source signatures
Checking if image destination supports signatures
Copying blob bb3c6092a4d4 done   |
Copying blob 45f3b16e76ca done   |
Copying blob 1185184a19a8 done   |
Copying blob c26e7813de54 done   |
Copying blob d75508b02042 done   |
Copying blob 7084cfa6f07d done   |
Copying config 801b74ee4a done   |
Writing manifest to image destination
Storing signatures
801b74ee4ac4e35bcc1645fdc3c1a56769b7d7963fdbf607d7e3000e9c140fcd
```

**Pull request merged**